src/utils/index.ts

```diff
@@ -13,7 +13,7 @@ const format = (str, ...args) => {
   return args.reduce((m, n) => m.replace('_s_', n), str);
 };
 
-export { t, format };
+export { format, t };
```

Comment:
can you remove code format changes like this? keep your PR simple and limit changes to manageable files. This will make review easier.